### PR TITLE
Iterations 1 to 3 By Cesar Jolibois

### PR DESCRIPTION
**Pull request by Diazblack**

### CODE REVIEW

File: lib/message.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    attr_reader :text,
+                :translator
+
+  def initialize(text, translator)
+    @text = text
+    @translator = translator
+  end
+
+  def translation
+    translator.translate(text)
```
Comment (memcmahon) [inline lib/message.rb:14]: When calling an instance variable from inside the class, we should use `@translator`.

File: lib/morse_translator.rb [hunk 2]
```diff
@@ -40,4 +41,21 @@ def initialize
                     " " => " "}
   end
 
+  def translate(message)
```
Comment (memcmahon) [inline lib/morse_translator.rb:44]: This is excellent!

File: lib/morse_translator.rb [hunk 3]
```diff
@@ -40,4 +41,21 @@ def initialize
...
+    translate = ""
+    message.each_char do |strings|
+      translate += dictionary[strings.downcase]
+    end
+    translate
+  end
+
+  def morse_to_eng(message)
```
Comment (memcmahon) [inline lib/morse_translator.rb:52]: SOOO close!  You just need to take into account the double spaces between words!

File: test/test_morse_translator.rb [hunk 4]
```diff
@@ -0,0 +1,75 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator.rb'
+
+class MorseTranslatorTest < Minitest::Test
```
Comment (memcmahon) [inline test/test_morse_translator.rb:5]: These tests look great!

**Review by memcmahon — COMMENTED** [review]
Cesar - this all looks excellent!

File: test/test_morse_translator.rb (hunk 4 above)
Comment (Diazblack) [reply test/test_morse_translator.rb:5]: Thanks Megan for the feedback. Have a great night

**Pull request closed without merging**